extensions/integration-tests/testfmks/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
This project could be moved out to its own module in a separate GitHub repo just like dataprotocols at a future time.